### DISCUSSION TITLE
Redash Extensions

### DISCUSSION THREAD

**Discussion by stephanie0butler** [getredash/redash#5917] [Q&A]
Hi all.
 
So how can I create an extension for a self-hosted redash instance? And how far can I go in extending the functionality of the Redash?
I have seen [the script](https://github.com/getredash/redash/blob/master/bin/bundle-extensions) which is not really informative. Also I have not found any documentation on this topic.

**Comment by arikfr** [comment]
We actually removed this extension point as it had very limited usage and was causing issues. You can still extend it, but need more details on what you want to do.
